libsolidity/interface/Natspec.cpp

```diff
@@ -178,12 +178,15 @@ Json::Value Natspec::extractReturnParameterDocs(std::multimap<std::string, DocTa
 			string paramName = _functionDef.returnParameters().at(n)->name();
 			string content = i->second.content;
 
-			if (paramName.empty())
+			auto nameEndPos = content.find_first_of(" \t");
```

Comment:
Question: can you give an example of the string content?

Reply:
the content could be `y value`, basically the part after `@return ` and it has to start with the name of the parameter